src/types/court-action-types.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  ActivateAnj: 'activateAnj',
```

Comment:
Ah realized that since we now updated the `activity-types` and `radspec` mappings to use the `action-types` (see https://github.com/aragon/court-dashboard/commit/adde4408ef74d244e4ea264fd899952858b7f982), if we change them to symbols then older activities that use the older convention would error.

Reply:
We decided to wipe older activities so we will update in favor of using symbols (see https://github.com/aragon/court-dashboard/pull/341)